### DISCUSSION TITLE
How to setup shareable organization config for 'bitbucket' platform?

### DISCUSSION THREAD

**Discussion by ablekhman** [renovatebot/renovate#8020] [Request Help]
We have self-hosted Renovate bot in our company. At the same time all our repos are hosted by Bitbucket Cloud. I cannot find a way to make [shareable configs](https://docs.renovatebot.com/config-presets/#organization-level-presets) for our repositories.

Even if I use specific prefix for the location of preset config, it is ignored and the platform is used instead.

Here is the snippet of the **config.js**:
```javascript
module.exports = {
    platform: 'bitbucket',
    ...
    onboardingConfig: {
        extends: [
            'github>username/renovate-config'
        ]
    }
}
```
The error message from the Renovate bot is: `"Unsupported platform 'bitbucket' for local preset."`

I tried replacing `local` with `bitbucket-server` or even `github`, but these locations are never checked. The error is always the same.

How can I share my configuration presets for Bitbucket Cloud platform?

If it is not supported, then how can I have configuration presets on a different platform from the target repos? Is publishing a npm package is the only option?

**Comment by viceice** [comment]
Local presets for Bitbucket cloud are currently not supported.

https://github.com/renovatebot/renovate/blob/f18b7f3504c3bdb799f1ff15fbe977cefb24641d/lib%2Fconfig%2Fpresets%2Flocal%2Findex.ts#L22-L39

**Reply by ablekhman** [reply]
@viceice , thanks for reply.

Is is possible to host configuration presets on a different platform from the one being scanned? I can keep configuration presets on Bitbucket Server or GitHub.

**Reply by viceice** [reply]
sure, checkout the [docs](https://docs.renovatebot.com/config-presets/#preset-hosting)

**Reply by ablekhman** [reply]
I have read this page. The reason I started this thread is that specifying any prefix for presets location has no effect. I'm not sure if it's a bug or a feature. The moment I have `bitbucket` as the configured platform, nothing else is working.

**Reply by viceice** [reply]
try this:
```js
onboardingConfig: {
    extends: [
        'github>username/renovate-config'
    ]
}
```

alternates are  `gitlab` and `gitea`.

They need to be hosted on github.com, gitlab.com or gitea.com and readable by the bot of cause.
So i would suggest to use github, as you maybe already have a github token configured for changelogs.

**Reply by ablekhman** [reply]
I tried exactly this. I updated my original post to reflect this fact and to avoid confusion for other readers. The Renovate bot ignores this prefix and reports the same error about `bitbucket` being the wrong host for presets.

**Reply by viceice** [reply]
Ok, this config works, please check if it's an **error** or **warning**. I 'm pretty sure you havn't checked the logs arround that message, which is a warning, so you can ignore it.

**Reply by ablekhman** [reply]
Yes, it is a warning. But the end result is that the config is not picked up. I can see it by the results of the scan.

**Reply by viceice** [reply]
`onboardingConfig` config is only used in onboarding branch, which isn't updated by default. You need to force a rebase of the onboarding pr, so renovate will update the onboarding pr.

If your repo is already onboarded, you sould add the preset manually to the repo, eg:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["github>username/renovate-config"]
}
```
